beets/library.py

```diff
@@ -494,6 +494,7 @@ class Item(LibModel):
         'mb_releasetrackid':    types.STRING,
         'trackdisambig':        types.STRING,
         'albumtype':            types.STRING,
+        'albumtypes':           types.STRING,
```

Comment:
I had to add this, because otherwise `beet import -A` would fail, because it sets values on an Item, iterating over `Album.item_keys`.

Reply:
Seems right to me!